easybuild/easyconfigs/t/TurboVNC/TurboVNC-2.2.3-GCCcore-8.2.0.eb

```diff
@@ -53,6 +53,11 @@ configopts += "-DOPENGL_egl_LIBRARY=$EBROOTLIBGLVND/lib/libEGL.%s " % SHLIB_EXT
 configopts += "-DOPENGL_opengl_LIBRARY=$EBROOTLIBGLVND/lib/libOpenGL.%s " % SHLIB_EXT
 configopts += "-DOPENGL_glx_LIBRARY=$EBROOTLIBGLVND/lib/libGLX.%s " % SHLIB_EXT
 
+# if installdir is under /opt, cmake will set SYSCONFDIR to /etc/installdir instead of installdir/etc
```

Comment:
The comment about what CMake will do is slightly wrong.
If installdir is under /opt, CMake will set SYSCONFDIR to /etc/opt/installdir

Reply:
you're right that the comment is inconsistent, but then it's the first part that is wrong, it should be "if installdir starts with /opt"

Reply:
Well both parts then. The text should perhaps be
"If installdir starts with /opt, i.e., /opt/xxx, CMake will set SYSCONFDIR to /etc/opt/xxx"
That way it's easily compared to the actual documentation from CMake and the FHS specification.